comparison.py

```diff
@@ -0,0 +1,334 @@
+    two methods of measurement. The mean of the two measurements is plotted on 
+    the x-axis, and the difference between the two measurements is plotted on the y-axis.
+
+    Parameters
+    ----------
+    data1, data2 : array-like
+        Arrays of data.
+    args, kwargs   
+        Other arguments are passed to the scatter plot function.
+
+    Returns
+    -------
+    None
+
+    Raises
+    ------
+    None
+
+    Notes
+    -----
+    The Bland-Altman plot is a simple and powerful way to visualize the agreement 
+    between two measurements. It is created by plotting the mean of two measurements 
+    against the difference between the two measurements. The mean is plotted on the x-axis, 
+    and the difference is plotted on the y-axis. The plot also includes the mean difference 
+    and the limits of agreement (mean difference ± 1.96 * standard deviation of the differences).
+    """
+    data1 = np.asarray(data1)
+    data2 = np.asarray(data2)
+    mean = np.mean([data1, data2], axis=0)  # Mean of the data
+    diff = data1 - data2  # Difference between data1 and data2
+    md = np.mean(diff)  # Mean of the difference
+    sd = np.std(diff, axis=0)  # Standard deviation of the difference
+
+    plt.scatter(mean, diff, *args, **kwargs)  # Use mean1 directly as x-values
+    plt.axhline(md, color='gray', linestyle='--', label=f'{md:.2f}')  # Mean Difference line
+    plt.axhline(md + 1.96 * sd, color='red', linestyle='--', label=f'{np.round(md + 1.96 * sd, decimals=3)}')
+    plt.axhline(md - 1.96 * sd, color='blue', linestyle='--', label=f'{np.round(md - 1.96 * sd, decimals=3)}')
+
+    # Add legend
+    plt.legend()
+    plt.xlabel('Mean of the two measurements')
+    plt.ylabel('Difference between the two measurements')
+
+def calculate_fundamental_component(signal, fs, low_f=0.66, high_f=3):
+    """
+    Calculate the fundamental component of a signal using the FFT method.
+    Parameters
+    ----------
+    signal : array-like
+        The input signal.
+    fs : int
+        The sampling frequency of the signal.
+    
+    Returns
+    -------
+    fund_f : float
+        The frequency of the fundamental component.
+    fund_amp : float
+        The amplitude of the fundamental component.
+    """
+    n_fft = len(signal)
+    win_fft_amp = sp_fft.rfft(sp_sig.detrend(signal) * sp_sig.windows.hann(n_fft), n=n_fft)
+    corr_factor = n_fft / np.sum(sp_sig.windows.hann(n_fft))
+    win_fft_amp = abs(win_fft_amp) * 2 / n_fft * corr_factor
+
+    win_fft_f = sp_fft.rfftfreq(n_fft, d=1 / fs)
+    f_low = int(low_f * n_fft / fs)
+    f_upp = int(high_f * n_fft / fs)
+    win_fft_amp_range = win_fft_amp[f_low:f_upp]
+    fund_idx = np.argmax(win_fft_amp_range) + f_low
+
+    fund_f = win_fft_f[fund_idx]
+    fund_amp = win_fft_amp[fund_idx]
+
+    return fund_f, fund_amp
+
+
+if __name__ == '__main__':
+    sampling_rate = int(os.getenv('SAMPLING_RATE'))
+    csv_files, name= os.getenv('PATH')

```

Comment:
Also rename PATH to for example DATA_PATH. PATH environment variable typically holds directories for executable files, not data directories.